collect_app/src/androidTest/java/org/odk/collect/android/regression/FillBlankFormTest.java

```diff
@@ -332,7 +333,7 @@ public void values_ShouldBeRandom() {
         for (int i = 1; i <= 3; i++) {
             FormEntryPage formEntryPage = new MainMenuPage().startBlankForm("random");
             firstQuestionAnswers.add(getQuestionText());
-            formEntryPage.swipeToNextQuestion();
+            formEntryPage.swipeToNextQuestion("random");
```

Comment:
Here question text is `Your random once value:`

Reply:
Yes, I see it but when I changed question text to `Your random once value:` I get `NoMatchingViewException`. I'm not sure why.

Reply:
Oh that's because the question label is dynamically generated by adding random value:
```
		<input ref="/random/rand_note">
			<label>Your random value: 
				<output value=" /random/rand "/></label>
		</input>
```
It would be good to introduce support for regex so that we could verify cases like this one. We can create a separate issue for that too.

Reply:
I've fixed this in https://github.com/getodk/collect/pull/4985/commits/38be1b2b95b09e26ed007ff2a66f9213693e7c8b
introducing:
```
private void assertQuestionText(String text) {
        onView(withIndex(withId(R.id.text_label), 0)).check(matches(withText(containsString(text))));
    }
```

so that now we are sure we test question label and we can verify dynamic labels passing just a part of it.